R/create_template.R

```diff
@@ -728,21 +728,21 @@ create_template <- function(
             "11_appendix.qmd"
           ),
           label = c(
-            "01_executive_summary",
-            "02_introduction",
-            "03_data",
-            "04a_assessment-configuration",
-            "04b_assessment-results",
-            "04c_assessment-sensitivity",
-            "04d_assessment-benchmarks",
-            "04e_assessment-projections",
-            "05_discussion",
-            "06_acknowledgments",
-            "07_references",
-            "08_tables",
-            "09_figures",
-            "10_notes",
-            "11_appendix"
+            "executive_summary",
+            "introduction",
+            "data",
+            "assessment-configuration",
+            "assessment-results",
+            "assessment-sensitivity",
+            "assessment-benchmarks",
+            "assessment-projections",
+            "discussion",
+            "acknowledgments",
+            "references",
+            "tables",
+            "figures",
+            "notes",
+            "appendix"
           )
         )
       } else {
```

Comment:
Nested if else statements like this are difficult to understand. For some, `dplyr::case_when()`, `dplyr::case_match()` or `switch()` statements can lead to code that is easier to follow and easier to add future cases to. E.g.,
```
x_option <- function(x) {
  switch(x,
    a = "option 1",
    b = "option 2",
    c = "option 3",
    stop("Invalid `x` value")
  )
}
```
which was copied from [Advanced R](https://adv-r.hadley.nz/control-flow.html).